scripts/busybox-ping-test-alcor-apis/01_one_subnet/00_vpm_default_table

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
```

Comment:
Oh I believe we will need to add MIT license statement to every new file as well.

Reply:
Added.